apps/docs/src/.vitepress/theme/theme.scss

```diff
@@ -19,3 +19,9 @@
     }
   }
 }
+
+.topic-overview {
```

Comment:
this is not used yet, intentionally?

Reply:
Yes the components will be used by Jannick in his PR. This is just the technical setup for it